koku/api/report/serializers.py

```diff
@@ -186,6 +194,35 @@ def _init_tag_keys(self, field, fargs=None, fkwargs=None):
             setattr(self, key, val)
             self.fields.update({key: val})
 
+    def _init_aws_category_keys(self, field, fargs=None, fkwargs=None):
```

Comment:
I might be missing something, but none of this belongs in this file. All of the AWS specific stuff should be moved to the aws serializers file:
https://github.com/project-koku/koku/blob/main/koku/api/report/aws/serializers.py